package.json

```diff
@@ -35,16 +37,16 @@
   },
   "dependencies": {
     "@edx/brand": "npm:@openedx/brand-openedx@^1.2.2",
-    "@edx/frontend-component-header": "^6.4.0",
-    "@edx/frontend-component-footer": "^14.6.0",
+    "@edx/frontend-component-header": "^8.0.0",
+    "@edx/frontend-component-footer": "^14.9.2",
     "@edx/frontend-platform": "^8.3.0",
     "@edx/openedx-atlas": "^0.7.0",
     "@fortawesome/fontawesome-svg-core": "1.2.36",
     "@fortawesome/free-brands-svg-icons": "5.15.4",
     "@fortawesome/free-regular-svg-icons": "5.15.4",
     "@fortawesome/free-solid-svg-icons": "5.15.4",
     "@fortawesome/react-fontawesome": "0.2.6",
-    "@openedx/paragon": "^23.4.5",
+    "@openedx/paragon": "^23.14.8",
     "core-js": "3.46.0",
     "prop-types": "15.8.1",
     "react": "17.0.2",
```

Comment:
I noticed that in the PR we’re transferring changes from, there was an upgrade of React and some other libraries.
Let’s bring those updates over as well and make sure everything works correctly.

https://github.com/openedx/frontend-app-catalog/pull/2/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R49

Reply:
The libraries that were removed or updated in [that PR](https://github.com/openedx/frontend-app-catalog/pull/2/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R49) should also be removed or updated in this one.